internal/acceptance/ip_access_list_test.go

```diff
@@ -4,21 +4,16 @@ import (
 	"testing"
 )
 
-func TestIPACLListsResourceFullLifecycle(t *testing.T) {
+func TestAccIPACLListsResourceFullLifecycle(t *testing.T) {
 	workspaceLevel(t, step{
 		Template: `
-		resource "databricks_workspace_conf" "features" {
-			enable_ip_access_lists = "true"
-		}
-		  
 		resource "databricks_ip_access_list" "this" {
```

Comment:
please add one more step, where you update this IP access list - e.g. by adding one more IP address range